fmriprep/cli/run.py

```diff
@@ -2,25 +2,20 @@
 # -*- coding: utf-8 -*-
 
 """
-fMRI run preprocessing workflow
+fMRI preprocessing workflow
 =====
 """
 
-import sys
-import os.path as op
-sys.path.append(op.dirname(__file__))
-import BIDSgenerator
-import glob
```

Comment:
I have for the most part un-done the `BIDSgenerator`-related changes. These are out-of-scope for this pull request. I think it might be a good idea to try to meet with your lab (I'm off campus, but there are others on-campus who could) to figure out how to get fmriprep to work with your data without changes to our core code, which are going to add friction to any pull requests.